nemo_skills/inference/prompt/llama3/eval_map.py

```diff
@@ -36,5 +36,8 @@
         'human-eval': 'llama3/codegen',
         'mbpp': 'llama3/codegen',
         'mmlu': 'llama3/mmlu',
+        # Math benchmarks
+        'gms8k': 'llama3/math_sft',
```

Comment:
Can you please update default to be math_sft and then explicitly set llama3/sft for ifeval/arena-hard instead? We have a ton of math benchmarks and it's better if we don't need to specify a specific prompt for each of them if we want to evaluate later